composer.json

```diff
@@ -11,6 +11,12 @@
       "role": "Developer"
     }
   ],
+  "repositories": {
+    "pimcore-editorconfig-bundle": {
+      "type": "git",
+      "url": "https://github.com/teamneusta/pimcore-editorconfig-bundle.git"
```

Comment:
Please make a version on packagist to avoid requiring git repositories on github bundle.

Reply:
The bundle is not ready yet... and just changed it's name ;)